### DISCUSSION TITLE
Responsive Data grid

### DISCUSSION THREAD

**Discussion by josfro** [microsoft/fluentui-blazor#1719] [Ideas]
Suggest making the Data grid component responsive to better display data on small screens. 

MudBlazor has done this - see examples below 

Example - wide screen 
![image](https://github.com/microsoft/fluentui-blazor/assets/80987290/36d85cde-8040-40e2-9a4b-d33b29be2f12)

Example - small screen 
![image](https://github.com/microsoft/fluentui-blazor/assets/80987290/60e36c6c-75f0-4d6b-96ed-555f74813348)

**Comment by vnbaaij** [comment]
We cant't do that currently because we use the datagrid web components for rendering the grid. These web components have not yet been optimized for displaying data in such a way.
